frontend/components/modal/ModalCommandPalette.vue

```diff
@@ -0,0 +1,196 @@
+              $t("components.modal-command-palette.jump-to")
+            }}</span>
+          </div>
+        </div>
+      </router-link>
+      <router-link to="/discussions">
+        <div class="flex mt-2">
+          <div>
+            <Icon
+              name="octicon:comment-discussion"
+              size="1em"
+              :alt="$t('_global.discussions')"
+            />
+          </div>
+          <div class="group command-palette-element-hover">
+            {{ $t("_global.discussions") }}
+            <span class="jump-link">{{
+              $t("components.modal-command-palette.jump-to")
+            }}</span>
+          </div>
+        </div>
+      </router-link>
+
+      <DialogTitle class="font-display flex justify-between mt-3">
+        <p class="text-3xl md:responsive-h2 font-bold">
+          {{ $t("_global.organization") }}
+        </p>
+      </DialogTitle>
+      <router-link to="/organizations">
+        <div class="flex mt-2">
+          <div>
+            <Icon
+              name="codicon:organization"
+              size="1em"
+              :alt="$t('components.modal-command-palette.organization-stub')"
+            />
+          </div>
+          <div class="group command-palette-element-hover">
+            {{ $t("components.modal-command-palette.organization-stub") }}
+            <span class="jump-link">{{
+              $t("components.modal-command-palette.jump-to")
+            }}</span>
+          </div>
+        </div>
+      </router-link>
+      <router-link to="/organizations">
+        <div class="flex mt-2">
+          <div>
+            <Icon
+              name="codicon:organization"
+              size="1em"
+              :alt="$t('components.modal-command-palette.organization-stub')"
+            />
+          </div>
+          <div class="group command-palette-element-hover">
+            {{ $t("components.modal-command-palette.organization-stub") }}
+            <span class="jump-link">{{
+              $t("components.modal-command-palette.jump-to")
+            }}</span>
+          </div>
+        </div>
+      </router-link>
+      <router-link to="/organizations">
+        <div class="flex mt-2">
+          <div>
+            <Icon
+              name="codicon:organization"
+              size="1em"
+              :alt="$t('components.modal-command-palette.organization-stub')"
+            />
+          </div>
+          <div class="group command-palette-element-hover">
+            {{ $t("components.modal-command-palette.organization-stub") }}
+            <span class="jump-link">{{
+              $t("components.modal-command-palette.jump-to")
+            }}</span>
+          </div>
+        </div>
+      </router-link>
+    </template>
+  </ModalBase>
+</template>
+
+<script setup>
+import { useMagicKeys, whenever } from "@vueuse/core";
+
+// TODO: Maybe there is a better / more future-proof solution than userAgent?
+const isMacOS = /Mac/.test(navigator.userAgent);
```

Comment:
You can use the device check module as is done [here](https://github.com/activist-org/activist/blob/cb57435f2fbb3e4262ca18e8b0404923d624526f/frontend/components/SearchBar.vue#L37C17-L37C17) 😊 These are also the lines we want to be uncommented :)

Reply:
This is as good of a solution as I could think of, and would be fine for now until we realize that it has major drawbacks.

Reply:
Got it. The $device.isMacOS (from the template example) doesn't work in a script block. It took me a minute, but I figured out to use useDevice().

Reply:
Awesome! Great to hear, @mattburnett-repo!